sandbox-server/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+
+app.use(bodyParser.urlencoded({ extended: false }))
+
+app.get('/', (request, response) => {
+  response.type('text/plain')
+  response.send('Welcome to Sandbox!')
+})
+
+app.get('/search', (request, response) => {
+  response.type('text/plain')
```

Comment:
I read the specs a bit differently from this implementation. I think it wants you to use the given search string in the response. I think you need to reject a non-doodads search